doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -54,6 +54,103 @@ This package contains various enum types useful for parsing and referring to imp
 Note that Move doesn't have enum types so unlike most of the above, these do not map directly to Move.
 However, each value in each enum is associated with a constant that exists in the Move code.
 
+## `econia_sdk.utils.decimals`
+
+This package contains a few helpers for calculating market parameters (lot size, tick size, and min size).
+The intent is to allow one to express their desired sizes in decimal notation, and have that converted to integer notation. These helpers make an effort to check for reasonable inputs, perfect results in all cases would be impossible due to integer division and decimal truncation.
+Price conversion utilities are left out for this reason: truncation, even a small percent's worth, would make them too dangerous.
```

Comment:
"Price conversion utilities are left out to avoid introducing truncation effects."